minkindr_gtsam/include/kindr/minimal/common-gtsam.h

```diff
@@ -8,4 +8,94 @@ typedef Eigen::Matrix<double, 3, 3> Jacobian3x3;
 typedef Eigen::Matrix<double, 6, 3> Jacobian6x3;
 typedef Eigen::Matrix<double, 6, 6> Jacobian6x6;
 
+namespace kindr {
+namespace minimal {
```

Comment:
This header is really unreadable. Can we split into a header and an implementation file?